pages/blog/graphql-deep-dive-1.mdx

```diff
@@ -0,0 +1,354 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: 'GraphQL - Diving Deep',
+  author: 'tvvignesh',
+  tags: ['graphql', 'deepdive', 'faq'],
+  date: '2020-11-28',
+  description:
+    'Answering all the questions you may have about GraphQL and its ecosystem',
+  image: '/blog-assets/graphql-deep-dive-1/cover.png',
+  thumbnail: '/blog-assets/graphql-deep-dive-1/cover.png',
+};
+
+export default withArticle({ ...meta });
+
+_This blog is a part of a series on GraphQL where we will dive deep into GraphQL and its ecosystem one piece at a time_
+
+The GraphQL specification was open sourced in 2015 by Facebook along with some basic implementations with a completely unique approach on how to structure, consume, transmit and process data and data graphs. It was a moment when a lot of people were convinced by its power, utility and promise that the rest became history.
+
+And today, there is a [**GraphQL foundation**](https://foundation.graphql.org/) which tries to ensure that GraphQL and the ecosystem thrives over time, [**a huge landscape of projects**](https://github.com/graphql/graphql-landscape) and [**this**](https://github.com/chentsulin/awesome-graphql) can be just one of the examples on how big the ecosystem has grown with a lot of languages, frameworks, tools supporting it as a first class citizen, so much so that even some of the [**huge enterprises**](https://graphql.org/users/) are using it today as part of their stack.
```

Comment:
Instead of awesome-graphql, which is not ordered and hard to look at, I would reference the new https://graphql.org/code/

Reply:
Sure. But I will add it as an additional link instead of removing `awesome-graphql` since I am not sure what projects are missing from either of the lists.